docker/gcloud_build.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+PROJECT=${PROJECT:?"gcloud project required"}
```

Comment:
If empty, can we use `gcloud config get-value project` to extract the current default project name from the gcloud config?

Reply:
OK, fixed it. PTAL.